dockers/docker-fpm-quagga/supervisord.conf

```diff
@@ -31,7 +31,7 @@ stdout_logfile=syslog
 stderr_logfile=syslog
 
 [program:zebra]
-command=/usr/lib/quagga/zebra -A 127.0.0.1
+command=/usr/lib/quagga/zebra -A 127.0.0.1 -s 2097152
```

Comment:
What is the default value? Is 2Mb good enough or should we increase it a bit more?

Reply:
Because we have system limit which is equal to this value. Check
https://github.com/Azure/sonic-buildimage/blob/201811/build_debian.sh#L364

Reply:
I see